test/cmd/wait.sh

```diff
@@ -94,6 +94,21 @@ EOF
     # Clean deployment
     kubectl delete deployment dtest
 
+    # create test data
+    kubectl create deployment test-3 --image=busybox
+
+    # wait with jsonpath without value to succeed
+    set +o errexit
+    # Command: Wait with jsonpath without value
+    output_message=$(kubectl wait --for=jsonpath='{.status.replicas}' deploy/test-3 2>&1)
```

Comment:
What would happen, if I run this test on master branch(without your patch)?
What would happen, if I run this command without `2>&1`?

Reply:
> What would happen, if I run this test on master branch(without your patch)?

The test will fail because the matching output will be different. It would say that the format is wrong and the JSON condition is missing a value.

> What would happen, if I run this command without `2>&1`?

I am not entirely sure because I do not know what output is redirected to `stderr`.